src/emulator/functionsEmulator.ts

```diff
@@ -647,6 +675,44 @@ export class FunctionsEmulator implements EmulatorInstance {
     }
   }
 
+  performPostLoadOperations(): Promise<void> {
+    if (
+      this.blockingFunctionsConfig.triggers?.beforeCreate?.functionUri === "" &&
+      this.blockingFunctionsConfig.triggers.beforeSignIn?.functionUri === ""
+    ) {
+      return Promise.resolve();
+    }
+
+    const authEmu = EmulatorRegistry.get(Emulators.AUTH);
+    if (!authEmu) {
+      return Promise.resolve();
+    }
```

Comment:
IIRC, functions emulator always starts before the auth emulator. Is there some mechanism to have the functions reloaded after auth emulator start or does it work in some other way?

Reply:
So the functions emulator gets the `start` method called before the auth emulator, but this code is executed in the `connect` stage which is always called after `start`, so we should be okay referencing auth here.